changelog.d/pr-7107.md

```diff
@@ -0,0 +1,65 @@
+<!-- Uncomment the section that is right (remove the HTML comment wrapper).-->
```

Comment:
Would be nice to delete the out commented stuff, since this is going to pile up in the repo over time.

Reply:
Good idea.

Reply:
done in df3bb1dba316b32904aa48e0185370a76f7d86e1